pkgs/applications/misc/flamerobin/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, wxGTK30, boost, firebird }:
+
+stdenv.mkDerivation rec {
+  version = "0.9.3.1";
+  name = "flamerobin-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/mariuz/flamerobin/archive/${version}.tar.gz";
+    sha256 = "06xiybgh9akln4ycspbzia29i1dazghg5k6znnwjdf7dab33hdgq";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = [ wxGTK30 boost firebird ];
+
+  preBuild = ''
+    sed -i 's/CXXFLAGS = -g -O2/CXXFLAGS = -g -O2 -nostartfiles/' Makefile
+  '';
+
+  configureFlags = [
+    "--disable-debug"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "FlameRobin is a database administration tool for Firebird RDBMS";
```

Comment:
It's better not to repeat the package name in the description to keep it short. And non-repetitive.